typings/src/ui/ui-events.ts

```diff
@@ -33,17 +33,29 @@ export enum EventTypes {
 
 export const enum EntityTypes {
   ENTRY = 0,
-  PROFILE,
-  COMMENT,
-  TAG,
+  PROFILE = 1,
+  COMMENT = 2,
+  TAG = 3,
+  ARTICLE = 4,
 }
 
+/**
+  To be used as per example `EntityTypesMap[EntityTypes.ENTRY]` to get the name of the entity type
+ */
+export const EntityTypesMap = {
+  [EntityTypes.ENTRY]: 'entry',
+  [EntityTypes.PROFILE]: 'profile',
+  [EntityTypes.COMMENT]: 'comment',
+  [EntityTypes.TAG]: 'tag',
+  [EntityTypes.ARTICLE]: 'article',
+} as const;
```

Comment:
having Entry as entry here and comment as comment conflicts with what you have in index.ts lines 79-87. If an example, maybe we could use the correct values too?